server/server/src/middleware/content_length_limit.rs

```diff
@@ -0,0 +1,98 @@
+use std::future::{ready, Ready};
+
+use actix_web::{
+    body::EitherBody,
+    dev::{forward_ready, Service, ServiceRequest, ServiceResponse, Transform},
+    http::StatusCode,
+    Error, HttpResponse,
+};
+use futures_util::future::LocalBoxFuture;
+use futures_util::TryStreamExt;
+
+static DEFAULT_MAX_UPLOAD_SIZE_BYTES: u64 = 100 * 1024 * 1024; // 100MB
```

Comment:
What should our default limit be? Should we make it configurable some how?

Reply:
I think yes, and 'not now' 😄 
When we have shifted admin to the central server we could have a configuration option?

Reply:
100MB is more than I would have allowed.. probably fine tho

Reply:
Yeah I thought surely that's big enough, but also not too restrictive...
PDQ PDF is 285MB, but I assume we'd upload parts to different assets not just one huge doc?

Reply:
some PDFs are massive! let's go with 100MB then 🙏